src/com/facebook/buck/cli/QuickstartCommand.java

```diff
@@ -71,6 +77,19 @@ public String getDestDir() {
   }
 
   public String getAndroidSdkDir() {
+    if (androidSdkDir == null) {
+      ProjectFilesystem projectFilesystem = new ProjectFilesystem(Paths.get("."));
+
+      AndroidDirectoryResolver androidDirectoryResolver =
+          new DefaultAndroidDirectoryResolver(projectFilesystem,
+              Optional.<String>absent(),
+              new DefaultPropertyFinder(projectFilesystem, ImmutableMap.copyOf(System.getenv())));
```

Comment:
As long as the daemon is inheriting the client's environment that should not be a problem, or?

Other than that, any hints on how to explicitly get the client's environment?